content/posts/2023-03-05-gather-town-building.md

```diff
@@ -0,0 +1,186 @@
+Slug: gather-town-building
+Authors: Winnie, Fish, Ray, Shirley, Vivian, Pochun, TN-Lee, Yoyo, Angus, Allen
+Summary: Last year (2022), PyCon Taiwan hosted PyCon APAC for the third time. Due to global epidemic situation, we chose holding PyCon APAC 2022 remotely via Gather.Town. It’s the second time we used Gather.Town to build the conference. We found that Gather.Town kept updating the interface and features, and created lots of friendly example maps for newcomers. In this article, we would introduce the events of PyCon APAC 2022 in the following.  
+
+## Introduction 
+
+Last year (2022), PyCon Taiwan hosted PyCon APAC for the third time. 
+
+Due to global epidemic situation, we chose holding PyCon APAC 2022 remotely via Gather.Town. It’s the second time we used Gather.Town to build the conference. We found that Gather.Town kept updating the interface and features, and created lots of friendly example maps for newcomers.
+
+In this article, we would introduce the events of PyCon APAC 2022 in the following.   <i> - written by Winnie - </i>
+
+<!--more-->
+
+## Overall
+#### Space A & Space B 
+
+We tried to introduce Taiwan street view in our main spaces, i.e. Space A & Space B. There is a temple in a rice cooker shape. In Taiwan, there are more than 2,200 temples, which exceeds the number of the convenience stores.  <i> - written by Fish - </i>
+
+<!-- ![Rice Cooker Temple](https://i.imgur.com/skTuzqm.jpg) -->
+![Rice Cooker Temple](images/2023-03-05-gather-town-building/0.jpg)
+> Rice Cooker Temple
+
+![The Sedar Chair](images/2023-03-05-gather-town-building/1.png)
+> The Sedar Chair
+
+Religious activities flourish in Taiwan. The temples hold shows and roadside banquets on their temple fairs. Hence, we put a big stage in Space A for the keynotes and lightning talks. There are also several red tables in Space B for open spaces. Both are common views on a temple fair.
+
+![The Stage in Space A](images/2023-03-05-gather-town-building/2.jpg)
+> The Stage in Space A
+
+![Red Tables in Space B](images/2023-03-05-gather-town-building/3.png)
+> Red Tables in Space B
+
+
+We also have common houses and the breakfast stores on the main spaces, which makes the foreign attendees feel like they are in Taiwan.
+
+![A Breakfast Store](images/2023-03-05-gather-town-building/4.png)
+> A Breakfast Store
+
+![A Plumber Store](images/2023-03-05-gather-town-building/5.png)
+> A Plumber Store
+
+#### PyNight 
+PyNight is the grand finale of PyCon TW’s first day. In this conventional event, the attendees could feel free to interact with others, accompanied by amazing musical performances and great food. This year, PyNight was hold in the form of BoFs (Birds of feather, flock together), which is a casual gathering for people with same interests. In the PyNight gather.town venue, the BoF spaces are designed in different styles: night camping, cyberpunk bar, concert hall and else.  <i> - written by Shirley - </i>
+
+![](images/2023-03-05-gather-town-building/6.jpg)
```

Comment:
Please add alternative text. Thanks!

from [mdlint](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md045---images-should-have-alternate-text-alt-text)
> Rationale: Alternate text is important for accessibility and describes the content of an image for people who may not be able to see it.

Reply:
Done. Thanks for your suggestion!